src/main/java/org/mvel2/asm/Opcodes.java

```diff
@@ -58,6 +58,7 @@
     int V1_6 = 0 << 16 | 50;
     int V1_7 = 0 << 16 | 51;
     int V1_8 = 0 << 16 | 52;
+    int V1_9 = 0 << 16 | 53;
```

Comment:
I was quickly checking the latest ASM release (5.1) and it does not contain the Opcodes.V1_9 and thus is not compatible with latest build of JDK 9 (which bumped the bytecode version). 

It seems that not even the master (trunk) is updated https://github.com/svn2github/asm-trunk/blob/master/asm/src/org/objectweb/asm/Opcodes.java.

So for now this custom patch seems like the best. We can then upgrade to next ASM once it supports JDK9. WDYT @mariofusco?